challenges/classes.js

```diff
@@ -1,7 +1,24 @@
 // 1. Copy and paste your prototype in here and refactor into class syntax.
+class CuboidMaker {
+    constructor(dimensions={}) {
+      this.length = dimensions.length;
+      this.width = dimensions.width;
+      this.height = dimensions.height;
+    }
+    
+    Volume(){
+        return this.length * this.width * this.height;
+    }
 
+    surfacArea(){
```

Comment:
Also note the spelling(/capitalization) errors here:
`surfacArea` should be `surfaceArea`
and
`Volume` should be `volume`
since that's how they're called down below